File: nl.naturalis.nba.client.php/model/Agent.php
<?php

namespace nl\naturalis\nba\client\php\model;

class Agent extends NBADomainObject {

	private $agentText;

	public function __construct()
	{
	}

	public function __construct($agentText)
	{
		$this->agentText = $agentText;
	}

	public function getAgentText()
	{
		return $this->agentText;
	}

	public function setAgentText(string $agentText)
	{
		$this->agentText = $agentText;
	}
}

File: nl.naturalis.nba.api/src/main/java/nl/naturalis/nba/api/model/Agent.java
package nl.naturalis.nba.api.model;

public class Agent extends NBADomainObject {

	private String agentText;

	public Agent()
	{
	}

	public Agent(String agentText)
	{
		this.agentText = agentText;
	}

	public String getAgentText()
	{
		return agentText;
	}

	public void setAgentText(String agentText)
	{
		this.agentText = agentText;
	}
}
